src/SFA.DAS.QnA.Application.UnitTests/ServiceTests/NotRequiredProcessorPagesWithDoesNotContainTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using SFA.DAS.QnA.Api.Types.Page;
+using SFA.DAS.QnA.Application.Services;
+
+namespace SFA.DAS.QnA.Application.UnitTests.ServiceTests
+{
+    [TestFixture]
+    public class NotRequiredProcessorPagesWithDoesNotContainTests
+    {
+        [TestCase("OrgType1", "", true, false, "detail without a toRemove in application data should render as satisfied NRC")]
+        [TestCase("", "OrgType2", false, true, "no details should not render as a satisfied NotRequiredCondition")]
+        [TestCase("OrgType2", "OrgType1", true, false, "not present so should be removed")]
+        [TestCase("OrgType1", "OrgType2", true, false, "not present so should be removed")]
+        [TestCase("OrgType1", "OrgType1", false, false, "present so should not be removed")]
+        [TestCase("rgType1", "OrgType1", true, false, "not present so should be removed")]
+        [TestCase("OrgType", "OrgType1", true, false, "not present so should be removed")]
+        [TestCase("orgType1", "OrgType1", false, false, "present so should not be removed")]
+
+      
+
+        public void When_PagesWithNotRequired_DoesNotContain_conditions_are_removed(string notRequiredConditionValue, string applicationDataValue, bool toRemove, bool singleValue, string explanation)
```

Comment:
Is a [Test] needed here?

Reply:
Yes, will raise story for tests

Reply:
there were testcase there, but I left 3 spaces by mistake....